rotkehlchen/globaldb/upgrades/v11_v12.py

```diff
@@ -13,6 +14,12 @@ def migrate_to_v12(connection: 'DBConnection', progress_handler: 'DBUpgradeProgr
     """This globalDB upgrade does the following:
     - Add new table for counterparty mappings
 
+    - Clears Velodrome/Aerodrome pool and gauge cache entries to ensure a full refresh.
+    Previously, fee and bribe addresses were not stored, so this forces an update
+    to include them where available.
```

Comment:
in general don't mix issues. Is this new? Why it happens? Where is it fixed?

Reply:
Okey I see, it was in the migration before and with the globaldb upgrade we just moved it